index.html

```diff
@@ -0,0 +1,184 @@
+<!-- Validated using html validator -->
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
Mixed tabs & spaces for indentation is 😢   You've used tabs here, but spaces throughout the rest of the document.  There's a setting in Sublime to auto-convert tabs to spaces that you can toss in your user preferences:
```"translate_tabs_to_spaces": true```